javalib/src/main/scala/java/util/IdentityHashMap.scala

```diff
@@ -18,15 +18,16 @@ import scala.annotation.tailrec
 
 import ScalaOps._
 
-class IdentityHashMap[K, V](inner: HashMap[IdentityHashMap.IdentityBox[K], V])
+class IdentityHashMap[K, V](
+    inner: HashMap[IdentityHashMap.IdentityBox[K], V], internal: Boolean)
```

Comment:
This should be documented as a workaround.

Also, why is this constructor not `private` / `private[java]` / ... if it is internal (I know it gets hidden away because this is only the binary interface, but still).

Reply:
Added a comment, and made the constructor `private`.